src/main/java/Deadlines.java

```diff
@@ -0,0 +1,43 @@
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class Deadlines extends Task {
+    private final String by;
```

Comment:
Perhaps can keep it as LocalDateTime to be more flexible, instead of having to parse every time